src/Compilers/CSharp/Portable/Syntax/Syntax.xml

```diff
@@ -3417,7 +3417,7 @@
       <Kind Name="SemicolonToken"/>
     </Field>
   </Node>
-  <Node Name="InterfaceDeclarationSyntax" Base="TypeDeclarationSyntax">
+  <Node Name="InterfaceDeclarationSyntax" Base="TypeDeclarationSyntax" SkipConvenienceFactories="true">
```

Comment:
>SkipConvenienceFactories="true" [](http://example.com/codeflow?start=71&length=31)

Should we add this to RecordDeclarationSyntax too?

Reply:
> Should we add this to RecordDeclarationSyntax too?

We are not making any changes for records, so no changes necessary, even if they wouldn't make any difference (which is unlikely).